.travis.yml

```diff
@@ -47,12 +63,14 @@ before_install:
 
 before_script:
   - mongo-orchestration start
-  - pushd ${MO_PATH} && ${TRAVIS_BUILD_DIR}/.travis.scripts/mo.sh configurations/servers/clean.json start && popd
+  - .travis.scripts/setup_mo.sh
   - ulimit -a
   - ulimit -c unlimited || true
   - .travis.scripts/before_script.sh
 
 script:
+  - export MONGODB_URI=`cat /tmp/uri.txt`
+  - echo $MONGODB_URI
```

Comment:
`setup_mo.sh` already prints the URI (prefixed with "MongoDB Test URI: "). Why bother doing so again?

Reply:
Originally, I added the echo here because for some reason the environment variable didn't propagate between steps (i.e., ``before_script`` and ``script``). I then later added the ``export`` to fix this, and the ``echo`` stayed for testing. I am inclined to keep it.